src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public static int selectRandomFigure() {
+        int minRandomNumber = 1;
+        int maxRandomNumber = 5; // maximum is limited by amount of figures we can choose from
```

Comment:
```suggestion
        int figureAmount = 5;
```
потрібно старатись писати код так, щоб коментарі були доцільні якумога рідше